system/security/zcrypt/zcrypt.cpp

```diff
@@ -153,18 +155,18 @@ int RSAZCryptor::setPublicKey(const char* publickey)
     return 0;
 }
 
-int RSAZCryptor::setPrivateKey(const char* privatekey, const char* passphrase)
+int RSAZCryptor::setPrivateKey(const char* privatekeyBuff, const char* passphrase)
 {
-    if(!privatekey || !*privatekey)
+    if(!privatekeyBuff || !*privatekeyBuff)
     {
         if(m_trace_level > 0)
-            printf("Warning: privatekey is empty\n");
+            printf("Warning: privatekeyBuff is empty\n");
 
         return -1;
     }
 
     if(m_trace_level > 10)
-        printf("setting privatekey:\n%s\n", privatekey);
```

Comment:
Is this something we really want to trace?

Reply:
Removed this and public key filename